jobs/config.json

```diff
@@ -10290,7 +10290,6 @@
       "--build=//...",
       "--install=gubernator/test_requirements.txt",
       "--test=//... //verify:verify-all",
```

Comment:
remove the comma :-) (it's the world of json

Reply:
done